Synthesis failed for dialogflow-python-client-v2

Hello! Autosynth couldn't regenerate dialogflow-python-client-v2. :broken_heart:

Here's the output from running `synth.py`:

```
0-07-31 05:12:35,169 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-07-31 05:12:35,190 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-07-31 05:12:35,234 autosynth [ERROR] > Synthesis failed
2020-07-31 05:12:35,234 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-07-31 05:12:35,239 autosynth [DEBUG] > Running: git checkout autosynth-googleapis
Switched to branch 'autosynth-googleapis'
2020-07-31 05:12:35,243 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-07-31 05:12:35,386 autosynth [DEBUG] > Running: git checkout 89d071c55c5bf9c7f766c5b49de7ce33b75222d6
Note: checking out '89d071c55c5bf9c7f766c5b49de7ce33b75222d6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-07-31 05:12:35,391 autosynth [DEBUG] > Running: git checkout 4d52dfb72078000b13de923c1dadec19f3a64ad1
HEAD is now at 4d52dfb7 fix: migrate accessapproval/v1 to grpc_service_config
2020-07-31 05:12:35,404 autosynth [DEBUG] > Running: git branch -f autosynth-512
2020-07-31 05:12:35,408 autosynth [DEBUG] > Running: git checkout autosynth-512
Switched to branch 'autosynth-512'
2020-07-31 05:12:35,412 autosynth [INFO] > Running synthtool
2020-07-31 05:12:35,412 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-07-31 05:12:35,412 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/512/sponge_log.log
2020-07-31 05:12:35,414 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-07-31 05:12:35,622 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-512
nothing to commit, working tree clean
2020-07-31 05:12:35,752 synthtool [DEBUG] > Ensuring dependencies.
2020-07-31 05:12:35,757 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Image is up to date for googleapis/artman:latest
2020-07-31 05:12:36,761 synthtool [DEBUG] > Cloning googleapis.
2020-07-31 05:12:36,762 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-07-31 05:12:36,768 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-07-31 05:12:36,811 autosynth [ERROR] > Synthesis failed
2020-07-31 05:12:36,811 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-07-31 05:12:36,816 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-31 05:12:36,820 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1b172ea4-6e2b-49f2-b1b9-cadea69c5d6e/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
08-01 05:12:52,438 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-01 05:12:52,462 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-01 05:12:52,526 autosynth [ERROR] > Synthesis failed
2020-08-01 05:12:52,526 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-01 05:12:52,532 autosynth [DEBUG] > Running: git checkout autosynth-googleapis
Switched to branch 'autosynth-googleapis'
2020-08-01 05:12:52,538 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-08-01 05:12:52,700 autosynth [DEBUG] > Running: git checkout 89d071c55c5bf9c7f766c5b49de7ce33b75222d6
Note: checking out '89d071c55c5bf9c7f766c5b49de7ce33b75222d6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-01 05:12:52,706 autosynth [DEBUG] > Running: git checkout 868615a5c1c1059c636bb3d82a555edb1d5a251e
HEAD is now at 868615a5 chore: move gaming to python microgenerator bazel rules
2020-08-01 05:12:52,722 autosynth [DEBUG] > Running: git branch -f autosynth-514
2020-08-01 05:12:52,727 autosynth [DEBUG] > Running: git checkout autosynth-514
Switched to branch 'autosynth-514'
2020-08-01 05:12:52,733 autosynth [INFO] > Running synthtool
2020-08-01 05:12:52,733 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-01 05:12:52,733 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/514/sponge_log.log
2020-08-01 05:12:52,736 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-01 05:12:52,976 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-514
nothing to commit, working tree clean
2020-08-01 05:12:53,133 synthtool [DEBUG] > Ensuring dependencies.
2020-08-01 05:12:53,140 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Image is up to date for googleapis/artman:latest
2020-08-01 05:12:54,217 synthtool [DEBUG] > Cloning googleapis.
2020-08-01 05:12:54,218 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-01 05:12:54,242 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-01 05:12:54,308 autosynth [ERROR] > Synthesis failed
2020-08-01 05:12:54,308 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-01 05:12:54,314 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-01 05:12:54,321 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d5a3b809-9df8-4d37-93b2-9714b500d6ce/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
08-02 05:12:35,546 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-02 05:12:35,566 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-02 05:12:35,615 autosynth [ERROR] > Synthesis failed
2020-08-02 05:12:35,615 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-02 05:12:35,703 autosynth [DEBUG] > Running: git checkout autosynth-googleapis
Switched to branch 'autosynth-googleapis'
2020-08-02 05:12:35,843 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-08-02 05:12:35,970 autosynth [DEBUG] > Running: git checkout 89d071c55c5bf9c7f766c5b49de7ce33b75222d6
Note: checking out '89d071c55c5bf9c7f766c5b49de7ce33b75222d6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-02 05:12:35,976 autosynth [DEBUG] > Running: git checkout 868615a5c1c1059c636bb3d82a555edb1d5a251e
HEAD is now at 868615a5 chore: move gaming to python microgenerator bazel rules
2020-08-02 05:12:36,004 autosynth [DEBUG] > Running: git branch -f autosynth-514
2020-08-02 05:12:36,008 autosynth [DEBUG] > Running: git checkout autosynth-514
Switched to branch 'autosynth-514'
2020-08-02 05:12:36,014 autosynth [INFO] > Running synthtool
2020-08-02 05:12:36,014 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-02 05:12:36,014 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/514/sponge_log.log
2020-08-02 05:12:36,017 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-02 05:12:36,270 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-514
nothing to commit, working tree clean
2020-08-02 05:12:36,418 synthtool [DEBUG] > Ensuring dependencies.
2020-08-02 05:12:36,425 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Image is up to date for googleapis/artman:latest
2020-08-02 05:12:37,509 synthtool [DEBUG] > Cloning googleapis.
2020-08-02 05:12:37,510 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-02 05:12:37,530 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-02 05:12:37,576 autosynth [ERROR] > Synthesis failed
2020-08-02 05:12:37,576 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-02 05:12:37,582 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-02 05:12:37,587 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a8e9e70d-f3c1-461c-bf82-b583dd8e94b6/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
08-03 05:12:37,065 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-03 05:12:37,087 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-03 05:12:37,134 autosynth [ERROR] > Synthesis failed
2020-08-03 05:12:37,134 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-03 05:12:37,140 autosynth [DEBUG] > Running: git checkout autosynth-googleapis
Switched to branch 'autosynth-googleapis'
2020-08-03 05:12:37,145 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-08-03 05:12:37,275 autosynth [DEBUG] > Running: git checkout 89d071c55c5bf9c7f766c5b49de7ce33b75222d6
Note: checking out '89d071c55c5bf9c7f766c5b49de7ce33b75222d6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-03 05:12:37,280 autosynth [DEBUG] > Running: git checkout 868615a5c1c1059c636bb3d82a555edb1d5a251e
HEAD is now at 868615a5 chore: move gaming to python microgenerator bazel rules
2020-08-03 05:12:37,294 autosynth [DEBUG] > Running: git branch -f autosynth-514
2020-08-03 05:12:37,299 autosynth [DEBUG] > Running: git checkout autosynth-514
Switched to branch 'autosynth-514'
2020-08-03 05:12:37,303 autosynth [INFO] > Running synthtool
2020-08-03 05:12:37,304 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-03 05:12:37,304 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/514/sponge_log.log
2020-08-03 05:12:37,306 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-03 05:12:37,527 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-514
nothing to commit, working tree clean
2020-08-03 05:12:37,667 synthtool [DEBUG] > Ensuring dependencies.
2020-08-03 05:12:37,673 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Image is up to date for googleapis/artman:latest
2020-08-03 05:12:38,780 synthtool [DEBUG] > Cloning googleapis.
2020-08-03 05:12:38,781 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-03 05:12:38,803 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-03 05:12:38,845 autosynth [ERROR] > Synthesis failed
2020-08-03 05:12:38,846 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-03 05:12:38,851 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-03 05:12:38,856 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9893a92c-09e8-4325-91b2-ae0125f1729e/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
020-08-04 05:12:44,282 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-04 05:12:44,301 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-04 05:12:44,346 autosynth [ERROR] > Synthesis failed
2020-08-04 05:12:44,347 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-04 05:12:44,352 autosynth [DEBUG] > Running: git checkout autosynth-googleapis
Switched to branch 'autosynth-googleapis'
2020-08-04 05:12:44,357 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-08-04 05:12:44,506 autosynth [DEBUG] > Running: git checkout 89d071c55c5bf9c7f766c5b49de7ce33b75222d6
Note: checking out '89d071c55c5bf9c7f766c5b49de7ce33b75222d6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-04 05:12:44,511 autosynth [DEBUG] > Running: git checkout dbdd8ddeb6d9556952aa8784d9e48f2566c9911a
HEAD is now at dbdd8dde replace html with markdown for accessapproval.proto
2020-08-04 05:12:44,524 autosynth [DEBUG] > Running: git branch -f autosynth-516
2020-08-04 05:12:44,528 autosynth [DEBUG] > Running: git checkout autosynth-516
Switched to branch 'autosynth-516'
2020-08-04 05:12:44,533 autosynth [INFO] > Running synthtool
2020-08-04 05:12:44,533 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-04 05:12:44,533 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/516/sponge_log.log
2020-08-04 05:12:44,535 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-04 05:12:44,753 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-516
nothing to commit, working tree clean
2020-08-04 05:12:44,893 synthtool [DEBUG] > Ensuring dependencies.
2020-08-04 05:12:44,898 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Image is up to date for googleapis/artman:latest
2020-08-04 05:12:45,885 synthtool [DEBUG] > Cloning googleapis.
2020-08-04 05:12:45,886 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-04 05:12:45,905 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-04 05:12:45,951 autosynth [ERROR] > Synthesis failed
2020-08-04 05:12:45,952 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-04 05:12:45,957 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-04 05:12:45,962 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/70b4b90d-addf-4885-8340-c6eebd84e430/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
t -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-05 05:11:41,552 autosynth [DEBUG] > Running: git checkout 6fd07563a2f1a6785066f5955ad9659a315e4492
Note: checking out '6fd07563a2f1a6785066f5955ad9659a315e4492'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 6fd07563 docs: change relative URLs to absolute URLs to fix broken links.
2020-08-05 05:11:41,564 autosynth [DEBUG] > Running: git branch -f autosynth-517
2020-08-05 05:11:41,567 autosynth [DEBUG] > Running: git checkout autosynth-517
Switched to branch 'autosynth-517'
2020-08-05 05:11:41,570 autosynth [INFO] > Running synthtool
2020-08-05 05:11:41,571 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-05 05:11:41,571 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/517/sponge_log.log
2020-08-05 05:11:41,574 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-05 05:11:41,768 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-517
nothing to commit, working tree clean
2020-08-05 05:11:41,901 synthtool [DEBUG] > Ensuring dependencies.
2020-08-05 05:11:41,906 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
ecc05c8a19c0: Waiting
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
5e2e46829c37: Waiting
98435f3e5753: Waiting
d60217dd00cb: Waiting
d33f0f205717: Waiting
1bb482c48abb: Waiting
69f904ad8fdb: Waiting
f80bef3513af: Waiting
4c0654f4dbfb: Waiting
44d6ab1173a5: Waiting
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
8e4cce5cf829: Pulling fs layer
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
1a1218157fc4: Pulling fs layer
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
60a2e61c889e: Waiting
db62096b2eb4: Waiting
90b72245d5fd: Waiting
b6e55165e523: Waiting
0018ab73693e: Waiting
25892f3ef9dd: Waiting
1ee837545391: Waiting
1e1e678ce8b0: Waiting
8e4cce5cf829: Waiting
7602d3a95dbd: Waiting
cdac5e33a980: Waiting
1a1218157fc4: Waiting
147a31960d92: Pulling fs layer
48e52bd78f71: Pulling fs layer
e4b1420d41e3: Pulling fs layer
147a31960d92: Waiting
9c7ffbad149d: Waiting
e4b1420d41e3: Waiting
4d77b1b29f2e: Verifying Checksum
4d77b1b29f2e: Download complete
7c793be88bae: Verifying Checksum
7c793be88bae: Download complete
ecc05c8a19c0: Download complete
7b378fa0f908: Verifying Checksum
7b378fa0f908: Download complete
5e2e46829c37: Verifying Checksum
5e2e46829c37: Download complete
d60217dd00cb: Verifying Checksum
d60217dd00cb: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
69f904ad8fdb: Verifying Checksum
69f904ad8fdb: Download complete
1bb482c48abb: Verifying Checksum
1bb482c48abb: Download complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
f80bef3513af: Download complete
7b378fa0f908: Pull complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
98435f3e5753: Verifying Checksum
98435f3e5753: Download complete
4d77b1b29f2e: Pull complete
7c793be88bae: Pull complete
ecc05c8a19c0: Pull complete
1e1e678ce8b0: Verifying Checksum
1e1e678ce8b0: Download complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
44d6ab1173a5: Verifying Checksum
44d6ab1173a5: Download complete
b6e55165e523: Verifying Checksum
b6e55165e523: Download complete
0018ab73693e: Verifying Checksum
0018ab73693e: Download complete
1ee837545391: Download complete
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
5e2e46829c37: Pull complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
1a1218157fc4: Verifying Checksum
1a1218157fc4: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
9c7ffbad149d: Verifying Checksum
9c7ffbad149d: Download complete
147a31960d92: Verifying Checksum
147a31960d92: Download complete
48e52bd78f71: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
25892f3ef9dd: Verifying Checksum
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
2020-08-05 05:12:18,935 synthtool [DEBUG] > Cloning googleapis.
2020-08-05 05:12:18,935 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-05 05:12:18,962 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-05 05:12:19,000 autosynth [ERROR] > Synthesis failed
2020-08-05 05:12:19,000 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-05 05:12:19,005 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-05 05:12:19,009 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4be218b6-dac7-413a-9b70-c61c140820cb/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-06 05:12:11,375 autosynth [DEBUG] > Running: git checkout 96ef637adf148d54236ad83fefa30c7f75f29737
Note: checking out '96ef637adf148d54236ad83fefa30c7f75f29737'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 96ef637a feat: Initial client generation for Dialogflow CX v3beta1.
2020-08-06 05:12:11,496 autosynth [DEBUG] > Running: git branch -f autosynth-520
2020-08-06 05:12:11,500 autosynth [DEBUG] > Running: git checkout autosynth-520
Switched to branch 'autosynth-520'
2020-08-06 05:12:11,505 autosynth [INFO] > Running synthtool
2020-08-06 05:12:11,505 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-06 05:12:11,505 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/520/sponge_log.log
2020-08-06 05:12:11,509 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-06 05:12:11,725 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-520
nothing to commit, working tree clean
2020-08-06 05:12:11,880 synthtool [DEBUG] > Ensuring dependencies.
2020-08-06 05:12:11,887 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
5e2e46829c37: Waiting
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
98435f3e5753: Waiting
d60217dd00cb: Waiting
d33f0f205717: Waiting
1bb482c48abb: Waiting
69f904ad8fdb: Waiting
f80bef3513af: Waiting
4c0654f4dbfb: Waiting
44d6ab1173a5: Waiting
ecc05c8a19c0: Waiting
db62096b2eb4: Waiting
1e1e678ce8b0: Waiting
b6e55165e523: Waiting
60a2e61c889e: Waiting
0018ab73693e: Waiting
8e4cce5cf829: Pulling fs layer
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
1a1218157fc4: Pulling fs layer
8e4cce5cf829: Waiting
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
147a31960d92: Pulling fs layer
90b72245d5fd: Waiting
48e52bd78f71: Pulling fs layer
e4b1420d41e3: Pulling fs layer
25892f3ef9dd: Waiting
7602d3a95dbd: Waiting
cdac5e33a980: Waiting
1a1218157fc4: Waiting
9c7ffbad149d: Waiting
48e52bd78f71: Waiting
e4b1420d41e3: Waiting
147a31960d92: Waiting
1ee837545391: Waiting
4d77b1b29f2e: Verifying Checksum
4d77b1b29f2e: Download complete
7c793be88bae: Verifying Checksum
7c793be88bae: Download complete
ecc05c8a19c0: Verifying Checksum
ecc05c8a19c0: Download complete
7b378fa0f908: Download complete
5e2e46829c37: Download complete
d60217dd00cb: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
69f904ad8fdb: Verifying Checksum
69f904ad8fdb: Download complete
1bb482c48abb: Verifying Checksum
1bb482c48abb: Download complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
f80bef3513af: Verifying Checksum
f80bef3513af: Download complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
98435f3e5753: Verifying Checksum
98435f3e5753: Download complete
7b378fa0f908: Pull complete
4d77b1b29f2e: Pull complete
7c793be88bae: Pull complete
ecc05c8a19c0: Pull complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
44d6ab1173a5: Verifying Checksum
44d6ab1173a5: Download complete
b6e55165e523: Verifying Checksum
b6e55165e523: Download complete
0018ab73693e: Verifying Checksum
1ee837545391: Verifying Checksum
1ee837545391: Download complete
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
5e2e46829c37: Pull complete
1e1e678ce8b0: Download complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
1a1218157fc4: Verifying Checksum
1a1218157fc4: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
9c7ffbad149d: Verifying Checksum
9c7ffbad149d: Download complete
147a31960d92: Verifying Checksum
147a31960d92: Download complete
48e52bd78f71: Verifying Checksum
48e52bd78f71: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
25892f3ef9dd: Verifying Checksum
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
2020-08-06 05:12:52,395 synthtool [DEBUG] > Cloning googleapis.
2020-08-06 05:12:52,396 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-06 05:12:52,420 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-06 05:12:52,468 autosynth [ERROR] > Synthesis failed
2020-08-06 05:12:52,468 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-06 05:12:52,474 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-06 05:12:52,479 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/45042024-fb19-4594-8181-0e5d4f28d63f/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
er) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-06 11:40:53,753 autosynth [DEBUG] > Running: git checkout 9ef5885643b012ea353c82ed81040d37c42e0206
Note: checking out '9ef5885643b012ea353c82ed81040d37c42e0206'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 9ef58856 Add Build bazel file to third party api folder
2020-08-06 11:40:53,828 autosynth [DEBUG] > Running: git branch -f autosynth-521
2020-08-06 11:40:53,832 autosynth [DEBUG] > Running: git checkout autosynth-521
Switched to branch 'autosynth-521'
2020-08-06 11:40:53,836 autosynth [INFO] > Running synthtool
2020-08-06 11:40:53,836 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-06 11:40:53,836 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/521/sponge_log.log
2020-08-06 11:40:53,840 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-06 11:40:54,045 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-521
nothing to commit, working tree clean
2020-08-06 11:40:54,186 synthtool [DEBUG] > Ensuring dependencies.
2020-08-06 11:40:54,190 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
8e4cce5cf829: Pulling fs layer
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
1a1218157fc4: Pulling fs layer
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
147a31960d92: Pulling fs layer
48e52bd78f71: Pulling fs layer
e4b1420d41e3: Pulling fs layer
b6e55165e523: Waiting
ecc05c8a19c0: Waiting
0018ab73693e: Waiting
5e2e46829c37: Waiting
1ee837545391: Waiting
98435f3e5753: Waiting
8e4cce5cf829: Waiting
90b72245d5fd: Waiting
d60217dd00cb: Waiting
25892f3ef9dd: Waiting
7602d3a95dbd: Waiting
d33f0f205717: Waiting
1a1218157fc4: Waiting
1bb482c48abb: Waiting
cdac5e33a980: Waiting
9c7ffbad149d: Waiting
147a31960d92: Waiting
48e52bd78f71: Waiting
e4b1420d41e3: Waiting
69f904ad8fdb: Waiting
db62096b2eb4: Waiting
4c0654f4dbfb: Waiting
44d6ab1173a5: Waiting
60a2e61c889e: Waiting
4d77b1b29f2e: Verifying Checksum
7c793be88bae: Verifying Checksum
7c793be88bae: Download complete
7b378fa0f908: Verifying Checksum
7b378fa0f908: Download complete
ecc05c8a19c0: Download complete
5e2e46829c37: Verifying Checksum
5e2e46829c37: Download complete
d60217dd00cb: Verifying Checksum
d60217dd00cb: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
69f904ad8fdb: Verifying Checksum
69f904ad8fdb: Download complete
1bb482c48abb: Verifying Checksum
1bb482c48abb: Download complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
f80bef3513af: Download complete
7b378fa0f908: Pull complete
4d77b1b29f2e: Pull complete
7c793be88bae: Pull complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
ecc05c8a19c0: Pull complete
98435f3e5753: Verifying Checksum
98435f3e5753: Download complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
1e1e678ce8b0: Verifying Checksum
1e1e678ce8b0: Download complete
44d6ab1173a5: Verifying Checksum
44d6ab1173a5: Download complete
0018ab73693e: Verifying Checksum
0018ab73693e: Download complete
b6e55165e523: Download complete
1ee837545391: Verifying Checksum
1ee837545391: Download complete
5e2e46829c37: Pull complete
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
1a1218157fc4: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
9c7ffbad149d: Verifying Checksum
9c7ffbad149d: Download complete
147a31960d92: Verifying Checksum
147a31960d92: Download complete
48e52bd78f71: Verifying Checksum
48e52bd78f71: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
25892f3ef9dd: Verifying Checksum
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
2020-08-06 11:41:32,351 synthtool [DEBUG] > Cloning googleapis.
2020-08-06 11:41:32,352 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-08-06 11:41:32,375 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-06 11:41:32,412 autosynth [ERROR] > Synthesis failed
2020-08-06 11:41:32,412 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-06 11:41:32,417 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-06 11:41:32,421 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f498856f-a16e-4b52-95db-7be3aea04b9f/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
 the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-07 05:12:00,685 autosynth [DEBUG] > Running: git checkout fb84629a56703d04f0b5304c4a9ade7313ebd92d
Note: checking out 'fb84629a56703d04f0b5304c4a9ade7313ebd92d'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at fb84629a chore: move bigquery connections to python microgenerator
2020-08-07 05:12:00,809 autosynth [DEBUG] > Running: git branch -f autosynth-524
2020-08-07 05:12:00,813 autosynth [DEBUG] > Running: git checkout autosynth-524
Switched to branch 'autosynth-524'
2020-08-07 05:12:00,818 autosynth [INFO] > Running synthtool
2020-08-07 05:12:00,818 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-07 05:12:00,818 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/524/sponge_log.log
2020-08-07 05:12:00,822 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-07 05:12:01,052 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-524
nothing to commit, working tree clean
2020-08-07 05:12:01,215 synthtool [DEBUG] > Ensuring dependencies.
2020-08-07 05:12:01,220 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
8e4cce5cf829: Pulling fs layer
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
1a1218157fc4: Pulling fs layer
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
5e2e46829c37: Waiting
147a31960d92: Pulling fs layer
48e52bd78f71: Pulling fs layer
e4b1420d41e3: Pulling fs layer
98435f3e5753: Waiting
ecc05c8a19c0: Waiting
1a1218157fc4: Waiting
cdac5e33a980: Waiting
d60217dd00cb: Waiting
9c7ffbad149d: Waiting
d33f0f205717: Waiting
1ee837545391: Waiting
60a2e61c889e: Waiting
147a31960d92: Waiting
48e52bd78f71: Waiting
db62096b2eb4: Waiting
8e4cce5cf829: Waiting
e4b1420d41e3: Waiting
90b72245d5fd: Waiting
1bb482c48abb: Waiting
25892f3ef9dd: Waiting
69f904ad8fdb: Waiting
7602d3a95dbd: Waiting
f80bef3513af: Waiting
4c0654f4dbfb: Waiting
44d6ab1173a5: Waiting
b6e55165e523: Waiting
0018ab73693e: Waiting
7c793be88bae: Verifying Checksum
7c793be88bae: Download complete
4d77b1b29f2e: Verifying Checksum
4d77b1b29f2e: Download complete
ecc05c8a19c0: Verifying Checksum
ecc05c8a19c0: Download complete
7b378fa0f908: Verifying Checksum
7b378fa0f908: Download complete
5e2e46829c37: Verifying Checksum
5e2e46829c37: Download complete
d60217dd00cb: Verifying Checksum
d60217dd00cb: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
69f904ad8fdb: Download complete
1bb482c48abb: Verifying Checksum
1bb482c48abb: Download complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
f80bef3513af: Verifying Checksum
f80bef3513af: Download complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
98435f3e5753: Verifying Checksum
98435f3e5753: Download complete
7b378fa0f908: Pull complete
4d77b1b29f2e: Pull complete
7c793be88bae: Pull complete
1e1e678ce8b0: Verifying Checksum
1e1e678ce8b0: Download complete
ecc05c8a19c0: Pull complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
44d6ab1173a5: Download complete
b6e55165e523: Download complete
0018ab73693e: Verifying Checksum
0018ab73693e: Download complete
1ee837545391: Verifying Checksum
1ee837545391: Download complete
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
5e2e46829c37: Pull complete
1a1218157fc4: Verifying Checksum
1a1218157fc4: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
9c7ffbad149d: Verifying Checksum
9c7ffbad149d: Download complete
48e52bd78f71: Verifying Checksum
48e52bd78f71: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
147a31960d92: Verifying Checksum
147a31960d92: Download complete
25892f3ef9dd: Verifying Checksum
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
2020-08-07 05:12:43,299 synthtool [DEBUG] > Cloning googleapis.
2020-08-07 05:12:43,300 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-07 05:12:43,359 autosynth [ERROR] > Synthesis failed
2020-08-07 05:12:43,359 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-07 05:12:43,365 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-07 05:12:43,370 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7b5df960-b6df-4f43-b5a7-3739cb841c68/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
s now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-08 05:11:55,708 autosynth [DEBUG] > Running: git checkout 3b11146ac64df40186e7f6e009dd3a13057b22cb
Note: checking out '3b11146ac64df40186e7f6e009dd3a13057b22cb'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 3b11146a uses generator option to override gapic module name
2020-08-08 05:11:55,721 autosynth [DEBUG] > Running: git branch -f autosynth-527
2020-08-08 05:11:55,724 autosynth [DEBUG] > Running: git checkout autosynth-527
Switched to branch 'autosynth-527'
2020-08-08 05:11:55,728 autosynth [INFO] > Running synthtool
2020-08-08 05:11:55,728 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-08 05:11:55,728 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/527/sponge_log.log
2020-08-08 05:11:55,732 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-08 05:11:55,957 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-527
nothing to commit, working tree clean
2020-08-08 05:11:56,107 synthtool [DEBUG] > Ensuring dependencies.
2020-08-08 05:11:56,112 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
8e4cce5cf829: Pulling fs layer
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
1a1218157fc4: Pulling fs layer
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
147a31960d92: Pulling fs layer
48e52bd78f71: Pulling fs layer
e4b1420d41e3: Pulling fs layer
ecc05c8a19c0: Waiting
b6e55165e523: Waiting
5e2e46829c37: Waiting
0018ab73693e: Waiting
1ee837545391: Waiting
7602d3a95dbd: Waiting
1a1218157fc4: Waiting
98435f3e5753: Waiting
cdac5e33a980: Waiting
d60217dd00cb: Waiting
d33f0f205717: Waiting
1bb482c48abb: Waiting
9c7ffbad149d: Waiting
69f904ad8fdb: Waiting
44d6ab1173a5: Waiting
f80bef3513af: Waiting
147a31960d92: Waiting
60a2e61c889e: Waiting
db62096b2eb4: Waiting
48e52bd78f71: Waiting
e4b1420d41e3: Waiting
4c0654f4dbfb: Waiting
8e4cce5cf829: Waiting
1e1e678ce8b0: Waiting
90b72245d5fd: Waiting
25892f3ef9dd: Waiting
4d77b1b29f2e: Verifying Checksum
4d77b1b29f2e: Download complete
7c793be88bae: Verifying Checksum
7c793be88bae: Download complete
ecc05c8a19c0: Download complete
7b378fa0f908: Verifying Checksum
7b378fa0f908: Download complete
5e2e46829c37: Verifying Checksum
5e2e46829c37: Download complete
d60217dd00cb: Verifying Checksum
d60217dd00cb: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
69f904ad8fdb: Verifying Checksum
69f904ad8fdb: Download complete
1bb482c48abb: Verifying Checksum
1bb482c48abb: Download complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
f80bef3513af: Verifying Checksum
f80bef3513af: Download complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
98435f3e5753: Verifying Checksum
98435f3e5753: Download complete
7b378fa0f908: Pull complete
4d77b1b29f2e: Pull complete
7c793be88bae: Pull complete
ecc05c8a19c0: Pull complete
1e1e678ce8b0: Verifying Checksum
1e1e678ce8b0: Download complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
44d6ab1173a5: Verifying Checksum
44d6ab1173a5: Download complete
b6e55165e523: Verifying Checksum
b6e55165e523: Download complete
0018ab73693e: Verifying Checksum
0018ab73693e: Download complete
1ee837545391: Verifying Checksum
1ee837545391: Download complete
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
5e2e46829c37: Pull complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
1a1218157fc4: Verifying Checksum
1a1218157fc4: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
9c7ffbad149d: Verifying Checksum
9c7ffbad149d: Download complete
147a31960d92: Verifying Checksum
147a31960d92: Download complete
48e52bd78f71: Verifying Checksum
48e52bd78f71: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
25892f3ef9dd: Verifying Checksum
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
2020-08-08 05:12:38,076 synthtool [DEBUG] > Cloning googleapis.
2020-08-08 05:12:38,077 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-08 05:12:38,121 autosynth [ERROR] > Synthesis failed
2020-08-08 05:12:38,121 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-08 05:12:38,126 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-08 05:12:38,130 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f99c3d93-31c8-48bc-bb19-84c0ff5cb272/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
y
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-09 05:11:45,860 autosynth [DEBUG] > Running: git checkout 3b11146ac64df40186e7f6e009dd3a13057b22cb
Note: checking out '3b11146ac64df40186e7f6e009dd3a13057b22cb'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 3b11146a uses generator option to override gapic module name
2020-08-09 05:11:45,938 autosynth [DEBUG] > Running: git branch -f autosynth-527
2020-08-09 05:11:45,942 autosynth [DEBUG] > Running: git checkout autosynth-527
Switched to branch 'autosynth-527'
2020-08-09 05:11:45,946 autosynth [INFO] > Running synthtool
2020-08-09 05:11:45,947 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-09 05:11:45,947 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/527/sponge_log.log
2020-08-09 05:11:45,950 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-09 05:11:46,164 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-527
nothing to commit, working tree clean
2020-08-09 05:11:46,308 synthtool [DEBUG] > Ensuring dependencies.
2020-08-09 05:11:46,313 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
8e4cce5cf829: Pulling fs layer
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
1a1218157fc4: Pulling fs layer
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
147a31960d92: Pulling fs layer
48e52bd78f71: Pulling fs layer
e4b1420d41e3: Pulling fs layer
ecc05c8a19c0: Waiting
5e2e46829c37: Waiting
4c0654f4dbfb: Waiting
d60217dd00cb: Waiting
44d6ab1173a5: Waiting
60a2e61c889e: Waiting
db62096b2eb4: Waiting
d33f0f205717: Waiting
1e1e678ce8b0: Waiting
b6e55165e523: Waiting
0018ab73693e: Waiting
1bb482c48abb: Waiting
f80bef3513af: Waiting
1ee837545391: Waiting
69f904ad8fdb: Waiting
7602d3a95dbd: Waiting
9c7ffbad149d: Waiting
8e4cce5cf829: Waiting
90b72245d5fd: Waiting
1a1218157fc4: Waiting
cdac5e33a980: Waiting
25892f3ef9dd: Waiting
48e52bd78f71: Waiting
e4b1420d41e3: Waiting
147a31960d92: Waiting
7c793be88bae: Verifying Checksum
7c793be88bae: Download complete
4d77b1b29f2e: Verifying Checksum
4d77b1b29f2e: Download complete
ecc05c8a19c0: Verifying Checksum
ecc05c8a19c0: Download complete
7b378fa0f908: Verifying Checksum
7b378fa0f908: Download complete
5e2e46829c37: Verifying Checksum
5e2e46829c37: Download complete
d60217dd00cb: Verifying Checksum
d60217dd00cb: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
69f904ad8fdb: Verifying Checksum
69f904ad8fdb: Download complete
1bb482c48abb: Verifying Checksum
1bb482c48abb: Download complete
f80bef3513af: Verifying Checksum
f80bef3513af: Download complete
7b378fa0f908: Pull complete
4d77b1b29f2e: Pull complete
98435f3e5753: Download complete
7c793be88bae: Pull complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
ecc05c8a19c0: Pull complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
1e1e678ce8b0: Verifying Checksum
1e1e678ce8b0: Download complete
44d6ab1173a5: Verifying Checksum
44d6ab1173a5: Download complete
b6e55165e523: Download complete
0018ab73693e: Verifying Checksum
0018ab73693e: Download complete
1ee837545391: Verifying Checksum
1ee837545391: Download complete
5e2e46829c37: Pull complete
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
1a1218157fc4: Verifying Checksum
1a1218157fc4: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
9c7ffbad149d: Download complete
147a31960d92: Verifying Checksum
147a31960d92: Download complete
48e52bd78f71: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
25892f3ef9dd: Verifying Checksum
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
2020-08-09 05:12:24,168 synthtool [DEBUG] > Cloning googleapis.
2020-08-09 05:12:24,169 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-09 05:12:24,228 autosynth [ERROR] > Synthesis failed
2020-08-09 05:12:24,228 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-09 05:12:24,233 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-09 05:12:24,238 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ff3112bf-d871-4db5-8af4-9eeff22dcf32/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
s now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-10 05:11:49,781 autosynth [DEBUG] > Running: git checkout 3b11146ac64df40186e7f6e009dd3a13057b22cb
Note: checking out '3b11146ac64df40186e7f6e009dd3a13057b22cb'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 3b11146a uses generator option to override gapic module name
2020-08-10 05:11:49,795 autosynth [DEBUG] > Running: git branch -f autosynth-527
2020-08-10 05:11:49,798 autosynth [DEBUG] > Running: git checkout autosynth-527
Switched to branch 'autosynth-527'
2020-08-10 05:11:49,802 autosynth [INFO] > Running synthtool
2020-08-10 05:11:49,802 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-10 05:11:49,802 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/527/sponge_log.log
2020-08-10 05:11:49,806 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-10 05:11:50,021 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-527
nothing to commit, working tree clean
2020-08-10 05:11:50,170 synthtool [DEBUG] > Ensuring dependencies.
2020-08-10 05:11:50,174 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
8e4cce5cf829: Pulling fs layer
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
1a1218157fc4: Pulling fs layer
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
69f904ad8fdb: Waiting
147a31960d92: Pulling fs layer
b6e55165e523: Waiting
f80bef3513af: Waiting
48e52bd78f71: Pulling fs layer
0018ab73693e: Waiting
4c0654f4dbfb: Waiting
1ee837545391: Waiting
e4b1420d41e3: Pulling fs layer
44d6ab1173a5: Waiting
cdac5e33a980: Waiting
60a2e61c889e: Waiting
8e4cce5cf829: Waiting
db62096b2eb4: Waiting
90b72245d5fd: Waiting
9c7ffbad149d: Waiting
25892f3ef9dd: Waiting
147a31960d92: Waiting
1e1e678ce8b0: Waiting
7602d3a95dbd: Waiting
e4b1420d41e3: Waiting
48e52bd78f71: Waiting
98435f3e5753: Waiting
d60217dd00cb: Waiting
1a1218157fc4: Waiting
d33f0f205717: Waiting
ecc05c8a19c0: Waiting
1bb482c48abb: Waiting
5e2e46829c37: Waiting
4d77b1b29f2e: Verifying Checksum
4d77b1b29f2e: Download complete
7c793be88bae: Download complete
ecc05c8a19c0: Verifying Checksum
ecc05c8a19c0: Download complete
7b378fa0f908: Verifying Checksum
7b378fa0f908: Download complete
5e2e46829c37: Verifying Checksum
5e2e46829c37: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
d60217dd00cb: Verifying Checksum
d60217dd00cb: Download complete
69f904ad8fdb: Verifying Checksum
69f904ad8fdb: Download complete
1bb482c48abb: Verifying Checksum
1bb482c48abb: Download complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
f80bef3513af: Verifying Checksum
f80bef3513af: Download complete
7b378fa0f908: Pull complete
98435f3e5753: Verifying Checksum
98435f3e5753: Download complete
4d77b1b29f2e: Pull complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
7c793be88bae: Pull complete
ecc05c8a19c0: Pull complete
1e1e678ce8b0: Verifying Checksum
1e1e678ce8b0: Download complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
44d6ab1173a5: Verifying Checksum
44d6ab1173a5: Download complete
b6e55165e523: Verifying Checksum
b6e55165e523: Download complete
0018ab73693e: Verifying Checksum
0018ab73693e: Download complete
1ee837545391: Verifying Checksum
1ee837545391: Download complete
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
5e2e46829c37: Pull complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
1a1218157fc4: Verifying Checksum
1a1218157fc4: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
9c7ffbad149d: Verifying Checksum
9c7ffbad149d: Download complete
147a31960d92: Verifying Checksum
147a31960d92: Download complete
48e52bd78f71: Verifying Checksum
48e52bd78f71: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
25892f3ef9dd: Verifying Checksum
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
2020-08-10 05:12:28,371 synthtool [DEBUG] > Cloning googleapis.
2020-08-10 05:12:28,372 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-10 05:12:28,429 autosynth [ERROR] > Synthesis failed
2020-08-10 05:12:28,430 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-10 05:12:28,434 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-10 05:12:28,439 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/bba8a73d-4eb6-4271-a445-069f24bdeaec/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
-b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-11 05:11:56,857 autosynth [DEBUG] > Running: git checkout 94006b3cb8d2fb44703cf535da15608eed6bf7db
Note: checking out '94006b3cb8d2fb44703cf535da15608eed6bf7db'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 94006b3c build: use gapic-generator-typescript v1.0.7.
2020-08-11 05:11:56,872 autosynth [DEBUG] > Running: git branch -f autosynth-531
2020-08-11 05:11:56,876 autosynth [DEBUG] > Running: git checkout autosynth-531
Switched to branch 'autosynth-531'
2020-08-11 05:11:56,881 autosynth [INFO] > Running synthtool
2020-08-11 05:11:56,881 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-08-11 05:11:56,881 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/dialogflow-python-client-v2/531/sponge_log.log
2020-08-11 05:11:56,885 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-08-11 05:11:57,107 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py.
On branch autosynth-531
nothing to commit, working tree clean
2020-08-11 05:11:57,256 synthtool [DEBUG] > Ensuring dependencies.
2020-08-11 05:11:57,261 synthtool [DEBUG] > Pulling artman image.
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
8e4cce5cf829: Pulling fs layer
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
1a1218157fc4: Pulling fs layer
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
147a31960d92: Pulling fs layer
48e52bd78f71: Pulling fs layer
e4b1420d41e3: Pulling fs layer
ecc05c8a19c0: Waiting
5e2e46829c37: Waiting
98435f3e5753: Waiting
d60217dd00cb: Waiting
1ee837545391: Waiting
d33f0f205717: Waiting
1bb482c48abb: Waiting
69f904ad8fdb: Waiting
f80bef3513af: Waiting
4c0654f4dbfb: Waiting
8e4cce5cf829: Waiting
44d6ab1173a5: Waiting
90b72245d5fd: Waiting
60a2e61c889e: Waiting
25892f3ef9dd: Waiting
db62096b2eb4: Waiting
7602d3a95dbd: Waiting
1a1218157fc4: Waiting
b6e55165e523: Waiting
0018ab73693e: Waiting
cdac5e33a980: Waiting
9c7ffbad149d: Waiting
147a31960d92: Waiting
e4b1420d41e3: Waiting
48e52bd78f71: Waiting
4d77b1b29f2e: Verifying Checksum
4d77b1b29f2e: Download complete
7c793be88bae: Verifying Checksum
7c793be88bae: Download complete
ecc05c8a19c0: Verifying Checksum
ecc05c8a19c0: Download complete
7b378fa0f908: Verifying Checksum
7b378fa0f908: Download complete
5e2e46829c37: Verifying Checksum
5e2e46829c37: Download complete
d60217dd00cb: Verifying Checksum
d60217dd00cb: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
69f904ad8fdb: Verifying Checksum
69f904ad8fdb: Download complete
1bb482c48abb: Verifying Checksum
1bb482c48abb: Download complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
f80bef3513af: Verifying Checksum
f80bef3513af: Download complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
98435f3e5753: Verifying Checksum
98435f3e5753: Download complete
7b378fa0f908: Pull complete
4d77b1b29f2e: Pull complete
7c793be88bae: Pull complete
ecc05c8a19c0: Pull complete
1e1e678ce8b0: Verifying Checksum
1e1e678ce8b0: Download complete
44d6ab1173a5: Verifying Checksum
44d6ab1173a5: Download complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
b6e55165e523: Verifying Checksum
b6e55165e523: Download complete
0018ab73693e: Verifying Checksum
0018ab73693e: Download complete
1ee837545391: Verifying Checksum
1ee837545391: Download complete
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
5e2e46829c37: Pull complete
1a1218157fc4: Verifying Checksum
1a1218157fc4: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
9c7ffbad149d: Verifying Checksum
9c7ffbad149d: Download complete
147a31960d92: Verifying Checksum
147a31960d92: Download complete
48e52bd78f71: Verifying Checksum
48e52bd78f71: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
2020-08-11 05:12:36,978 synthtool [DEBUG] > Cloning googleapis.
2020-08-11 05:12:36,979 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-11 05:12:37,047 autosynth [ERROR] > Synthesis failed
2020-08-11 05:12:37,048 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
2020-08-11 05:12:37,053 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-11 05:12:37,058 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/196f8a04-9215-4923-887e-d554ded632e7/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

Autosynth is still having trouble generating dialogflow-python-client-v2. :sob:

Here's the output from running `synth.py`:

```
2020-08-12 05:12:27,425 autosynth [INFO] > logs will be written to: /tmpfs/src/logs/dialogflow-python-client-v2
Switched to branch 'autosynth'
Note: checking out '89d071c55c5bf9c7f766c5b49de7ce33b75222d6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
Note: checking out '43a62a87b70010d9cf9be31e99ea230a535e1b47'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 43a62a87 migrate dataproc python BUILD file to use microgen
Switched to branch 'autosynth-539'
2020-08-12 05:12:32,833 autosynth [INFO] > Running synthtool
2020-08-12 05:12:32,834 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
On branch autosynth-539
nothing to commit, working tree clean
latest: Pulling from googleapis/artman
7b378fa0f908: Pulling fs layer
4d77b1b29f2e: Pulling fs layer
7c793be88bae: Pulling fs layer
ecc05c8a19c0: Pulling fs layer
5e2e46829c37: Pulling fs layer
98435f3e5753: Pulling fs layer
d60217dd00cb: Pulling fs layer
d33f0f205717: Pulling fs layer
1bb482c48abb: Pulling fs layer
69f904ad8fdb: Pulling fs layer
f80bef3513af: Pulling fs layer
4c0654f4dbfb: Pulling fs layer
44d6ab1173a5: Pulling fs layer
60a2e61c889e: Pulling fs layer
db62096b2eb4: Pulling fs layer
1e1e678ce8b0: Pulling fs layer
b6e55165e523: Pulling fs layer
0018ab73693e: Pulling fs layer
1ee837545391: Pulling fs layer
ecc05c8a19c0: Waiting
5e2e46829c37: Waiting
8e4cce5cf829: Pulling fs layer
98435f3e5753: Waiting
90b72245d5fd: Pulling fs layer
25892f3ef9dd: Pulling fs layer
7602d3a95dbd: Pulling fs layer
d60217dd00cb: Waiting
d33f0f205717: Waiting
1a1218157fc4: Pulling fs layer
cdac5e33a980: Pulling fs layer
9c7ffbad149d: Pulling fs layer
147a31960d92: Pulling fs layer
48e52bd78f71: Pulling fs layer
e4b1420d41e3: Pulling fs layer
1bb482c48abb: Waiting
8e4cce5cf829: Waiting
f80bef3513af: Waiting
9c7ffbad149d: Waiting
4c0654f4dbfb: Waiting
25892f3ef9dd: Waiting
147a31960d92: Waiting
44d6ab1173a5: Waiting
60a2e61c889e: Waiting
1a1218157fc4: Waiting
0018ab73693e: Waiting
cdac5e33a980: Waiting
db62096b2eb4: Waiting
1ee837545391: Waiting
1e1e678ce8b0: Waiting
b6e55165e523: Waiting
7c793be88bae: Verifying Checksum
7c793be88bae: Download complete
4d77b1b29f2e: Download complete
ecc05c8a19c0: Verifying Checksum
ecc05c8a19c0: Download complete
7b378fa0f908: Verifying Checksum
7b378fa0f908: Download complete
5e2e46829c37: Verifying Checksum
5e2e46829c37: Download complete
d33f0f205717: Verifying Checksum
d33f0f205717: Download complete
d60217dd00cb: Verifying Checksum
d60217dd00cb: Download complete
69f904ad8fdb: Verifying Checksum
69f904ad8fdb: Download complete
1bb482c48abb: Download complete
4c0654f4dbfb: Verifying Checksum
4c0654f4dbfb: Download complete
f80bef3513af: Verifying Checksum
f80bef3513af: Download complete
60a2e61c889e: Verifying Checksum
60a2e61c889e: Download complete
98435f3e5753: Verifying Checksum
98435f3e5753: Download complete
1e1e678ce8b0: Verifying Checksum
1e1e678ce8b0: Download complete
db62096b2eb4: Verifying Checksum
db62096b2eb4: Download complete
7b378fa0f908: Pull complete
44d6ab1173a5: Verifying Checksum
44d6ab1173a5: Download complete
4d77b1b29f2e: Pull complete
7c793be88bae: Pull complete
b6e55165e523: Verifying Checksum
b6e55165e523: Download complete
0018ab73693e: Verifying Checksum
0018ab73693e: Download complete
ecc05c8a19c0: Pull complete
1ee837545391: Verifying Checksum
90b72245d5fd: Verifying Checksum
90b72245d5fd: Download complete
7602d3a95dbd: Verifying Checksum
7602d3a95dbd: Download complete
1a1218157fc4: Verifying Checksum
1a1218157fc4: Download complete
cdac5e33a980: Verifying Checksum
cdac5e33a980: Download complete
5e2e46829c37: Pull complete
9c7ffbad149d: Verifying Checksum
9c7ffbad149d: Download complete
8e4cce5cf829: Verifying Checksum
8e4cce5cf829: Download complete
48e52bd78f71: Verifying Checksum
48e52bd78f71: Download complete
147a31960d92: Download complete
e4b1420d41e3: Verifying Checksum
e4b1420d41e3: Download complete
25892f3ef9dd: Verifying Checksum
25892f3ef9dd: Download complete
98435f3e5753: Pull complete
d60217dd00cb: Pull complete
d33f0f205717: Pull complete
1bb482c48abb: Pull complete
69f904ad8fdb: Pull complete
f80bef3513af: Pull complete
4c0654f4dbfb: Pull complete
44d6ab1173a5: Pull complete
60a2e61c889e: Pull complete
db62096b2eb4: Pull complete
1e1e678ce8b0: Pull complete
b6e55165e523: Pull complete
0018ab73693e: Pull complete
1ee837545391: Pull complete
8e4cce5cf829: Pull complete
90b72245d5fd: Pull complete
25892f3ef9dd: Pull complete
7602d3a95dbd: Pull complete
1a1218157fc4: Pull complete
cdac5e33a980: Pull complete
9c7ffbad149d: Pull complete
147a31960d92: Pull complete
48e52bd78f71: Pull complete
e4b1420d41e3: Pull complete
Digest: sha256:0da97ae2ac274adc8ee147fd1eb6736f8186d1684b0f58bce91f4295797a5dcd
Status: Downloaded newer image for googleapis/artman:latest
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/dialogflow-python-client-v2/synth.py", line 26, in <module>
    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 48, in py_library
    return self._generate_code(service, version, "python", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_generator.py", line 119, in _generate_code
    f"Unable to find configuration yaml file: {(googleapis / config_path)}."
FileNotFoundError: Unable to find configuration yaml file: /home/kbuilder/.cache/synthtool/googleapis/google/cloud/dialogflow/v2beta1/artman_dialogflow_v2beta1.yaml.
2020-08-12 05:13:19,845 autosynth [ERROR] > Synthesis failed
HEAD is now at 89d071c docs: move to googleapisdotdev (#155)
Switched to branch 'autosynth'
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/40aebe43-ce34-44d8-a657-65fcbc6bcdb6/targets/github%2Fsynthtool;config=default/tests;query=dialogflow-python-client-v2;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -17,13 +17,18 @@
 import synthtool as s
 from synthtool import gcp
 
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICBazel()
 
 versions = ["v2beta1", "v2"]
 
 
 for version in versions:
-    library = gapic.py_library("dialogflow", version,  config_path=version + '/artman_dialogflow_' + version + '.yaml', include_protos=True)
+    library = gapic.py_library(
+        "dialogflow",
+        version,
+        bazel_target=f"//google/cloud/dialogflow/{version}:dialogflow-{version}-py",
+        include_protos=True,
+    )
 
     s.move(
         library,
@@ -95,29 +100,26 @@ s.replace(
 )
 
 # Docstring has '-----' which is interpreted as RST section title
-s.replace(
-    "dialogflow_v2beta1/proto/intent_pb2.py",
-    "\s+-----------",
-    ""
-)
+s.replace("dialogflow_v2beta1/proto/intent_pb2.py", "\s+-----------", "")
 
 s.replace(
     "dialogflow_*/proto/session_pb2.py",
     "============================================================================",
-    ""
+    "",
 )
 
 
 # Replace bad hyperlink references
-s.replace(
-    "dialogflow_*/proto/audio_config_pb2.py",
-    "\s*\<\>`__",
-    "`"
-)
+s.replace("dialogflow_*/proto/audio_config_pb2.py", "\s*\<\>`__", "`")
 
 
-s.replace('dialogflow_v2/proto/agent_pb2.py', ':math:', '')
-s.replace('dialogflow_v2/proto/agent_pb2.py', ':raw-latex:', '')
+s.replace("dialogflow_v2/proto/agent_pb2.py", ":math:", "")
+s.replace("dialogflow_v2/proto/agent_pb2.py", ":raw-latex:", "")
+
+# ignore sphinx warnings
+# TODO: remove with microgenerator transition
+s.replace("noxfile.py",
+"""['"]-W['"],  # warnings as errors""",
+"")
 
 s.shell.run(["nox", "-s", "blacken"], hide_output=False)
-

